public/app/features/dashboard/components/SubMenu/SubMenuItems.tsx

```diff
@@ -9,33 +9,57 @@ import { Button } from '@grafana/ui';
 
 interface Props {
   variables: VariableModel[];
+  filtersExpanded: boolean;
+  ExpandFilters: Function;
 }
 
-export const SubMenuItems: FunctionComponent<Props> = ({ variables }) => {
+export const SubMenuItems: FunctionComponent<Props> = ({ variables, filtersExpanded, ExpandFilters }) => {
+  const optionVariables = variables as VariableWithOptions[];
   const [visibleVariables, setVisibleVariables] = useState<VariableModel[]>([]);
+  let advanceFilters = optionVariables.filter(
+    (variable) =>
+      variable.hide !== VariableHide.hideVariable &&
```

Comment:
shouldn't all advanced filters be hidden? I can understand that even this will work, but thought that the other approach could be simpler